From 73e7a79e98454499606553aabb7860c8505d9b99 Mon Sep 17 00:00:00 2001
From: Favour Ejiofor <faveejiofor2009@gmail.com>
Date: Sat, 20 Jun 2026 04:09:12 +0100
Subject: [PATCH 5/5] Remove env from tracking

---
 .env | 7 -------
 1 file changed, 7 deletions(-)
 delete mode 100644 .env

diff --git a/.env b/.env
deleted file mode 100644
index 62fc1ce..0000000
--- a/.env
+++ /dev/null
@@ -1,7 +0,0 @@
-VITE_SUPABASE_URL=https://wrgtdnouvahimgsioisa.supabase.co
-VITE_SUPABASE_ANON_KEY=sb_publishable_iGjC1uK0fPctewW5S9RQew_49RnNLz5
-VITE_PAYSTACK_PUBLIC_KEY=pk_test_9d2eaa4ecacb0bedee203310d3b4141d70626c41
-VITE_BASIC_PACKAGE_AMOUNT_KOBO=300000
-VITE_PRO_PACKAGE_AMOUNT_KOBO=400000
-VITE_PAYSTACK_CURRENCY=NGN
-VITE_SERVER_URL=http://localhost:5000
\ No newline at end of file
-- 
2.54.0.windows.1

